lib/services/event_service.ex

```diff
@@ -108,25 +105,52 @@ defmodule Aprb.Service.EventService do
           unfurl_links: true
          }
       "conversations" ->
-        if event["verb"] == "buyer_outcome_set" && event["properties"]["buyer_outcome"] == "other" do
-          %{
-            text: ":phone: #{event["subject"]["display"]} responded on https://www.artsy.net/artwork/#{List.first(event["properties"]["conversation_items"])["item_id"]}",
-            attachments: "[{
-                            \"fields\": [
-                              {
-                                \"title\": \"Outcome\",
-                                \"value\": \"#{event["properties"]["buyer_outcome"]}\",
-                                \"short\": true
-                              },
-                              {
-                                \"title\": \"Comment\",
-                                \"value\": \"#{event["properties"]["buyer_outcome_comment"]}\",
-                                \"short\": false
-                              }
-                            ]
-                          }]",
-            unfurl_links: true
-          }
+        case event["verb"] do
+          "buyer_outcome_set" ->
+            if event["properties"]["buyer_outcome"] == "other" do
+              %{
+                text: ":phone: #{event["subject"]["display"]} responded on https://www.artsy.net/artwork/#{List.first(event["properties"]["conversation_items"])["item_id"]}",
```

Comment:
I think that could still work. You'd pass in the item id and the helper would construct the URL with it.